dspace-api/src/main/java/org/dspace/app/statistics/clarin/ClarinMatomoBitstreamTracker.java

```diff
@@ -137,8 +154,14 @@ public void trackBitstreamDownload(Context context, HttpServletRequest request,
             return;
         }
 
-        // Log the user which is downloading the bitstream
-        this.logUserDownloadingBitstream(context, bit);
+        if (!isZip) {
+            // Log the user which is downloading the bitstream
+            this.logUserDownloadingBitstream(context, bit);
+        } else {
+            // Track the zip file downloading event
+            this.logUserDownloadingZip(context, item);
+        }
+
         // Track the bitstream downloading event
         trackPage(context, request, item, "Bitstream Download / Single File");
```

Comment:
even zip is bitstream download/single file?

Reply:
You are right! It should be `Bitstream Download / Zip Archive` thanks.